spec/registry.md

```diff
@@ -826,3 +812,24 @@ For more information, see [Working with Timezones](https://w3c.github.io/timezon
 > The form of these serializations is known and is a de facto standard.
 > Support for these extensions is expected to be required in the post-tech preview.
 > See: https://datatracker.ietf.org/doc/draft-ietf-sedate-datetime-extended/
+
+### Date and Time Locale Options
+
+Implementations MAY support overriding some of the options set by the current locale
+or an implementation-defined date/time _operand_ value.
+To do so, the following **_<dfn>date/time locale options</dfn>_**
+can be made available on `:datetime`, `:date`, and `:time`:
+
+- `calendar`
+  - valid [Unicode Calendar Identifier](https://cldr-smoke.unicode.org/spec/main/ldml/tr35.html#UnicodeCalendarIdentifier)
+- `numberingSystem`
+   - valid [Unicode Number System Identifier](https://cldr-smoke.unicode.org/spec/main/ldml/tr35.html#UnicodeNumberSystemIdentifier)
+- `hour12`
+  - `true`
+  - `false`
+- `timeZone`
+  - valid identifier per [BCP175](https://www.rfc-editor.org/rfc/rfc6557)
```

Comment:
I think we need to do a bit more here. The ability to float and unfloat time values is actually rather important, even if developers don't always understand it. See: [Working with Time and Timezones](https://w3c.github.io/timezone/#floating-times)

```suggestion
- `timeZone` **optional** (no default)
  - valid identifier per [BCP175](https://www.rfc-editor.org/bcp/bcp175)
  - `none` (removes the time zone, if present, to create a local/floating time)
```